content/docs/deployment/csminstallationwizard/src/index.html

```diff
@@ -331,6 +343,18 @@
                     </div>
                   </div>
 
+                  <div class="was-validated row mb-4 fsGroupPolicy">
+                    <label for="fsGroupPolicy" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Select the fsGroupPolicy type" style="width: 180px;">FSGroupPolicy Type</label>
+                    <div class="col-sm-9"></div>
+                    <div class="col-sm-3">
+                      <select id="fsGroup-Policy" class="form-select" onchange="validateInput(validateForm, CONSTANTS)" required>
+                        <option value="" selected> Select the FSGroupPolicy type</option>
+                        <option value="ReadWriteOnceWithFSType">ReadWriteOnceWithFSType</option>
```

Comment:
Can we make ReadWriteOnceWithFSType to be selected by default if it is the default value?

Reply:
Updated the code, Please check